doc/source/structures/misc/config.rst

```diff
@@ -127,11 +127,15 @@ Configuration of kOS
 
     Configures the ``EnableSafeMode`` setting.
 
+.. highlight:: none
+
     If true, then it enables the following error messages::
```

Comment:
Same as below. Sphinx thinks that 'If true, then it enables' belongs to highlight and throws an error.